src/input/src/Input.tsx

```diff
@@ -1019,6 +1018,7 @@ export default defineComponent({
                 readonly={this.readonly as any}
                 autofocus={this.autofocus}
                 size={this.attrSize}
+                {...this.inputProps}
```

Comment:
I think props property has higher priority than inputProp like autofocus

Reply:
I don't think inputProps should has higher priority, since it may break default behavior without any warning.